packages/gatsby/src/db/__tests__/node-tracking-test.js

```diff
@@ -89,7 +88,19 @@ describe(`track root nodes`, () => {
     let type
 
     beforeAll(async () => {
-      type = (await nodeTypes.buildAll({})).testNode.nodeObjectType
+      const { SchemaComposer } = require(`graphql-compose`)
```

Comment:
Considering this is a such common pattern, is our API a bit wrong? Ideally we would be able to call `addInferredType({ schemaComposer, typeName: 'Test', nodeStore: fakeNodeStore, })`, what is preventing that from happening?

Reply:
Oh this would work perfectly, if we provide a nodeStore